TradeItIosTicketSDK2/TradeItBrokerManagementViewController.swift

```diff
@@ -14,12 +17,39 @@ class TradeItBrokerManagementViewController: TradeItViewController, TradeItBroke
 
         self.brokerManagementTableManager.delegate = self
         self.brokerManagementTableManager.brokersTable = self.brokersTableView
+        
+        self.activityView = MBProgressHUD.showAdded(to: self.view, animated: true)
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        self.brokerManagementTableManager.updateLinkedBrokers(withLinkedBrokers: TradeItSDK.linkedBrokerManager.linkedBrokers)
+        self.activityView?.label.text = "Authenticating"
+        TradeItSDK.linkedBrokerManager.authenticateAll(
```

Comment:
@guillaumedebavelaere @mitochondrion when are we going to change this to `authenticateAllIfNeeded`?